src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -9,6 +9,8 @@ partial class BoundStatement : IStatement
     {
         OperationKind IOperation.Kind => this.StatementKind;
 
+        bool IOperation.HasErrors => this.HasErrors;
```

Comment:
Why do we need this?  This seems like an extra unnecessary indirectoin.  Why not just have the existing HasErrors implement IOperation.HasErrors instead?